### PR TITLE
Add booking created mailer

### LINKED ISSUE #8: Create mailers and jobs after reserved rooms
- [x] Send a confirmation email to the user after a new booking is created
  - [x] Send an email to the hotel email

### PR DESCRIPTION
**Pull request by MariethPerez**
Add booking created mailer
Send email to user after a booking was created
Send email to hotel after a booking was created
Resolve #8

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -5,4 +5,12 @@ class Booking < ApplicationRecord
   def self.available?(min_date, max_date)
     where('end_date >= ? AND start_date <= ?', min_date, max_date)
   end
+end
+  after_create :notification_booking_created
```
Comment (condef5) [inline app/models/booking.rb]: Hey fixed these code.

Comment (MariethPerez) [reply app/models/booking.rb]: I will fix in the next pull request.

File: app/models/booking.rb [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
   def self.available?(min_date, max_date)
     where('end_date >= ? AND start_date <= ?', min_date, max_date)
   end
+
+  def notification_booking_created
+    BookingMailer.with(booking: self).booking_created_user.deliver_now
+    BookingMailer.with(booking: self).booking_created_hotel.deliver_now
+  end
```
Comment (liannivin) [inline app/models/booking.rb]: Hey Marieth. It's better that you use deliver_later instead deliver_now.

Comment (MariethPerez) [reply app/models/booking.rb]: I will fix in the next pull request.

**Pull request merged**